README.md

```diff
@@ -8,3 +8,20 @@ Useful links:
 - [NPM package](https://www.npmjs.com/package/notifications-node-client)
 - [Changelog](https://github.com/alphagov/notifications-node-client/blob/master/CHANGELOG.md)
 - [Contributing to this client](https://github.com/alphagov/notifications-node-client/blob/master/CONTRIBUTING.md)
+
+## Testing JavaScript examples in Markdown
```

Comment:
Should look at how to make this run as part of continuous integration.

Reply:
I have added this to the 'npm test' command which runs automatically.

Reply:
Yeah, noticed straight after I posted that.

***

Also, this section should go in `CONTRIBUTING.md` not `README.md` (we separate out guidance that isn’t relevant to people consuming the client by putting it in there).